src/rgtk.rs

```diff
@@ -60,8 +60,9 @@ extern crate "cairo-sys" as cairo_ffi;
 
 extern crate glib;
 
+pub use glib::Type;
+pub use glib::{GValue, GValuePublic};
```

Comment:
Hmm, if you're doing this for convenience of the example, maybe just reexport `glib` instead of cherry-picking its types? I seemed to see an attribute that wouldn't let its documentation spill into this crate's when doing that...